.github/workflows/ci-for-default-ns.yaml

```diff
@@ -75,15 +75,21 @@ jobs:
           # for testing start-debug and stop-debug deployment
           # mons
           kubectl rook_ceph debug start rook-ceph-mon-a
-          sleep 5
+          sleep 10
+          debug_mon=$(kubectl get pod -l app=rook-ceph-mon,app.kubernetes.io/instance=a -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
+          timeout 1000 sh -c 'until [ "$(kubectl get pod $debug_mon -n rook-ceph -o jsonpath='{.status.phase}'| grep -c "Running")" -eq 1 ]; do echo "waiting for pod rook-ceph-mon-a-debug to get created " && sleep 1; done'
```

Comment:
@travisn the CI is again and again failing here,

I am not sure why this is happeening, 
I increased the timeout to 1000s still it didn't came into ready state

Reply:
Did you confirm the pod is correctly retrieved on line 79? Maybe the pod is not being found. For debugging i'd suggest breaking these long lines apart and print intermediate results.

Reply:
Logs

```
deployment.apps/rook-ceph-mon-a-debug created
+ sleep 10
++ kubectl get pod -l app=rook-ceph-mon,app.kubernetes.io/instance=a -n rook-ceph -o 'jsonpath={.items[*].metadata.name}'
+ debug_mon=rook-ceph-mon-a-debug-58774f6fb6-xjnp6
+ timeout 1000 sh -c 'until [ "$(kubectl get pod $debug_mon -n rook-ceph -o jsonpath={.status.phase}| grep -c "Running")" -eq 1 ]; do echo "waiting for pod rook-ceph-mon-a-debug to get created " && sleep 1; done'
waiting for pod rook-ceph-mon-a-debug to get created 
waiting for pod rook-ceph-mon-a-debug to get created 
```


Yes it got created and fetched properly,

In timeout I runn the same command it run fine